frontend/app/src/locales/en.json

```diff
@@ -73,10 +73,12 @@
     },
     "frontend": {
       "label": {
+        "query_period": "Periodic client query period in seconds",
```

Comment:
Periodic client query period sounds a bit repetive. Wonder if it can be rephrased a bit better both here and in line 139-140.

Something like `"Client querying period"` would also probably explain it sufficiently?